### DISCUSSION TITLE
Translation bug in Portuguese, Brazil

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#3483] [Translations]
### Describe your idea

At the midweek meeting, after adding media, the name 'Add Media' appears. I believe this is a translation error, and as shown in English, the ideal would be for 'MEDIA ADDED' or 'EXTRA MEDIA' to appear after the media is added.

### Additional context

![image](https://github.com/user-attachments/assets/7a86c65b-4378-4ea6-a29b-6e384017cc6b)

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my idea is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my idea is not a duplicate.

**Comment by joaochris** [comment]
Sim, foi um erro na tradução. 
Foi corrigido, obrigado!
